user/configure-device.rst

```diff
@@ -0,0 +1,132 @@
+    enable SSH access and connect the VM to the internet.
+
+3. Install openwisp-config on your LEDE instance
+------------------------------------------------
+
+Installation
+~~~~~~~~~~~~
+
+.. note::
+
+    An installation guide is also available
+    `here <https://github.com/openwisp/openwisp-config#install-precompiled-package>`__.
+
+Run the following commands on your device:
+
+.. code-block:: bash
+
+    opkg update
+    opkg install <URL>
+
+Replace ``<URL>`` with the link to one of the latest build, available
+`here <http://downloads.openwisp.org/openwisp-config/latest/>`__. For
+this guide, let's choose the ``openssl`` package, but any of them should
+work the same.
+
+Configuration
+~~~~~~~~~~~~~
+
+Once it's installed, we need to configure it to connect to our OpenWISP2
+controller. To do that, edit the config file located at
+``/etc/config/openwisp``.
+
+You will see the default config file, something like the following:
+
+::
+
+    # For more information about these config options please see the README
+    # or https://github.com/openwisp/openwisp-config#configuration-options
+
+    config controller 'http'
+        #option url 'https://openwisp2.mynetwork.com'
+        #option interval '120'
+        #option verify_ssl '1'
+        #option shared_secret ''
+        #option consistent_key '1'
+        #option mac_interface 'eth0'
+        #option merge_config '1'
+        #option test_config '1'
+        #option test_script '/usr/sbin/mytest'
+        option uuid ''
+        option key ''
+        list unmanaged 'system.@led'
+        list unmanaged 'network.loopback'
+        list unmanaged 'network.@switch'
+        list unmanaged 'network.@switch_vlan'
+        # curl options
+        #option connect_timeout '15'
+        #option max_time '30'
+        #option capath '/etc/ssl/certs'
+
+Uncomment and change the following fields:
+
+-  ``url``: The hostname of your OpenWISP2 controller
+-  ``verify_ssl``: Set to ``0`` if your controller's SSL certificate is
+   self-signed
+-  ``shared_secret``: Get it from OpenWISP2's admin panel, located at
+   the Group settings
+
+Save the file and start openwisp-config:
+
+.. code-block:: bash
+
+    /etc/init.d/openwisp_config start
+
+Your LEDE instance should register itself to your openwisp2 controller.
+Check the devices menu on the admin panel to make sure your LEDE
+instance is registered.
+
+Compiling your own LEDE/OpenWRT image
+-------------------------------------
+
+You may want to compile a custom LEDE/OpenWRT image to save time on configuring new devices. By compiling a custom image, you
+can pre-install openwisp-config, including your configurations (e.g. `shared_secret`), so that you wouldn't have to go through the
+configuration process again. This will be very useful if you want to manage many devices.
+
+A guide on how to do it is available `here <https://github.com/openwisp/openwisp-config#compiling-a-custom-lede--openwrt-image>`_.
```

Comment:
excellent. I just noticed that you are not breaking lines at the same character length of the rest of the document, could you make it consistent?